src/common/actions/repositories.js

```diff
@@ -67,9 +68,15 @@ export function fetchUserRepositories(pageNumber) {
           if (result.pageLinks) {
             dispatch(setPageLinks(result.pageLinks));
           }
+        })
+        .catch(error => {
+          return Promise.reject(error);
         });
       })
-      .catch(error => dispatch(fetchRepositoriesError(error)));
+      .catch(error => {
+        log.warn(error);
```

Comment:
@earnubs: They won't be. This was a run of the mill development bug I encountered while I was working on a component file. I forgot to define `styles`. The surprising bit that this error was swallowed by a promise you'd expect was unrelated 😩